src/select/select.js

```diff
@@ -402,16 +402,24 @@ class Select extends React.Component<PropsT, SelectStateT> {
     }
   };
 
-  getOptionLabel = ({
-    option,
-  }: {
-    option: OptionT,
-    optionState: {
-      $selected: boolean,
-      $disabled: boolean,
-      $isHighlighted: boolean,
+  getOptionLabel = (
+    locale: LocaleT,
+    {
+      option,
+    }: {
+      option: OptionT,
+      optionState: {
+        $selected: boolean,
+        $disabled: boolean,
+        $isHighlighted: boolean,
+      },
     },
-  }): React.Node => {
+  ): React.Node => {
+    if (option.isCreatable) {
+      return (
+        locale.select.create + ' ' + '"' + option[this.props.labelKey] + '"'
```

Comment:
would a template literal be more readable here?

Reply:
Yea but it somehow kills code highlighting in my VS Code 😞